example/src/main/java/com/stripe/example/activity/LaunchPaymentSheetCompleteActivity.kt

```diff
@@ -47,7 +58,30 @@ internal class LaunchPaymentSheetCompleteActivity : BasePaymentSheetActivity() {
                 onSuccess = { json ->
                     val clientSecret = json.getString("secret")
 
-                    onPaymentIntent(
+                    onIntentCreated(
```

Comment:
if you make onSuccess into a function it seems it could be shared between setup and payment intent cases

Reply:
This changed now, PI and SI must call different methods.